### ISSUE TITLE
Uncaught TypeError: this.editor.getDecreaseNextIndentPattern is not a function

### ISSUE THREAD

**Issue by danqing** [DSpeckhals/python-indent#51]
[Enter steps to reproduce:]

1. Just trying to break up lines.

```py
annotations = _xxx_xxxx_xxxxxxxxxx_to_xxxxxx(yyyyy_yyy,
                            xxxxx_xxxx_xxxx, xxxxxxx_sql)
```

was trying to hit enter before `yyyyy_yyy`.

**Atom**: 1.22.0-beta0 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.6
**Thrown From**: [python-indent](https://github.com/DSpeckhals/python-indent) package 1.1.1

### Stack Trace

Uncaught TypeError: this.editor.getDecreaseNextIndentPattern is not a function

```
At /Users/dliu/.atom/packages/python-indent/lib/python-indent.js:299

TypeError: this.editor.getDecreaseNextIndentPattern is not a function
    at PythonIndent.indentHanging (/packages/python-indent/lib/python-indent.js:299:51)
    at PythonIndent.indent (/packages/python-indent/lib/python-indent.js:34:18)
    at HTMLElement.editorNewline (/packages/python-indent/lib/main.js:23:57)
    at CommandRegistry.handleCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app/src/command-registry.js:381:36)
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:621:22)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:412:28)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom Beta.app/Contents/Resources/app/src/window-event-handler.js:100:42)
    at HTMLDocument.<anonymous> (/Applications/Atom Beta.app/Contents/Resources/app/src/window-event-handler.js:3:65)
```

### Commands

```
     -0:28.4.0 core:backspace (input.hidden-input)
     -0:28 editor:move-to-end-of-line (input.hidden-input)
  2x -0:27.8.0 core:move-left (input.hidden-input)
     -0:26.8.0 editor:newline (input.hidden-input)
     -0:26.6.0 core:backspace (input.hidden-input)
     -0:26.3.0 core:save (input.hidden-input)
     -0:24.6.0 editor:consolidate-selections (input.hidden-input)
     -0:24.6.0 core:cancel (input.hidden-input)
     -0:23.2.0 core:cut (input.hidden-input)
     -0:23 core:save (input.hidden-input)
     -0:19.8.0 core:cut (input.hidden-input)
     -0:19.6.0 core:save (input.hidden-input)
     -0:07.8.0 editor:newline (input.hidden-input)
     -0:07.4.0 snippets:next-tab-stop (input.hidden-input)
     -0:07.4.0 snippets:expand (input.hidden-input)
     -0:07.4.0 editor:indent (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.30.5 
atom-ide-ui 0.4.0 
autocomplete-python 1.10.3 
busy-signal 1.4.3 
file-icons 2.1.12 
filesize 3.0.0 
git-blame 1.3.1 
go-debug 1.5.2 
go-plus 5.5.5 
go-signature-statusbar 1.2.3 
hyperclick 0.0.0 
ide-python 0.2.1 
ide-typescript 0.6.1 
import-sf-mono 1.0.1 
indent-guide-improved 1.4.13 
intentions 1.1.5 
language-diff 0.7.0 
language-gettext 0.6.1 
language-jade 0.7.2 
language-nginx 0.8.0 
language-protobuf 0.7.1 
linter 2.2.0 
linter-eslint 8.3.2 
linter-flake8 2.2.1 
linter-jsonlint 1.3.0 
linter-sass-lint 1.8.2 
linter-ui-default 1.6.10 
MagicPython 1.0.12 
merge-conflicts 1.4.5 
pigments 0.40.2 
pristine-ui 1.1.10 
python-indent 1.1.1 
python-tools 0.6.9 
react 0.16.2 
sort-lines 0.15.0 
uglify 1.3.1 
```

**Comment by DSpeckhals** [comment]
It looks like this is affecting the beta version of Atom only. An API was moved to a different context, and has to be changed. I'll see if I can get a fix out soon.

**Comment by DSpeckhals** [comment]
Thanks for catching this issue in the beta channel. It helps to find these types of bugs before most people every see them.

I just published version 1.1.2 of the package with the fix.
